src/test/java/teammates/test/cases/webapi/EnrollStudentsActionTest.java

```diff
@@ -86,7 +118,8 @@ public void testExecute_withSectionFieldChanged_shouldBeUpdatedToDatabase() {
         List<StudentData> enrolledStudents = executeActionAndReturnResults(courseId, req);
 
         assertEquals(1, enrolledStudents.size());
-        verifyStudentInDatabase(studentToUpdate, enrolledStudents.get(0).getCourseId(), enrolledStudents.get(0).getEmail());
+        verifyStudentInDatabase(
```

Comment:
Well, there is the same new line character problem here

Reply:
My bad. Fixed